_includes/abort-reference.md

```diff
@@ -67,6 +66,9 @@ Content-Type: application/json
         "payeeInfo": {
             "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/payeeinfo"
         },
+        "payers": {
+           "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/payers"
+        },
```

Comment:
Are we certain `payers` are available for most merchants?

Reply:
Have sent this to Nina and Øyvind, awaiting answers.

Reply:
Still waiting on confirmation.

Reply:
What did you find out about the availability of `payers`?

Reply:
See above :) Still waiting.

Reply:
After a talk with Nina, it seems like the answer is yes.